github/github.go

```diff
@@ -212,6 +212,9 @@ type ListCursorOptions struct {
 
 	// A cursor, as given in the Link header. If specified, the query only searches for events before this cursor.
 	Before string `url:"before,omitempty"`
+
+	// A cursor, as given in the Link header. If specified, the query continues the search using this cursor.
+	Cursor string `url:"cursor,omitempty"`
```

Comment:
I believe this is the first GitHub API that uses `Cursor` for pagination. And I've added it to `ListCursorOptions` rather than `ListOptions` as... the name seemed to state that it wants all the possible list options for GitHub APIs that support pagination.

Please tell me if you'd like a brand new struct like `ListHookDeliveriesOptions`. Honestly, I'd personally prefer that as the Webhook Deliveries API as of today doesn't support other fields defined in ListCursorOptions. But go-github's current design looked more inclined to have a fewer number of common structs, rather than having a lot of specific structs. That's why I've chosen to change ListCursorOptions rather than having a new struct.

Reply:
I think adding it to `ListCursorOptions` is fine.  Thanks.